proto/blockchain.proto

```diff
@@ -189,6 +191,10 @@ message DescribeChain {
     repeated string supportedSubscriptions = 8;
 }
 
+message BuildInfo {
+    string version = 1;
```

Comment:
maybe add more fields?
It is not prohibited to use your own fork of dshackle for example

Reply:
we discussed that this can be done later, when we realize that it is necessary